### PR TITLE
smart styling

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@Rizziepit @SteveBarnett pls review

File: springboard_iogt/templates/base.jinja2 [hunk 1]
```diff
@@ -39,14 +40,13 @@
...
-        <ul>
+        <div class="h1"><div class="detail">{% trans %}Explore IoGT{% endtrans %}</div></div>
+        <ul class="options">
         {% for section in content_sections %}
             {% set section_url=content_section_url_overrides.get(section.slug, 'content_section'|route_url(slug=section.slug)) %}
-            <li><a href="{{ section_url }}">{{ section.title }}</a>
+            <li class="{{section.slug}}"><a href="{{ section_url }}">{{ section.title }}<p>{{section.descriptor}}</p></a>
             </li>
         {% endfor %}
-        <li><a href="{{ 'home'|route_url }}">{{ _("Home") }}</a></li>
```
Comment (Rizziepit) [inline springboard_iogt/templates/base.jinja2:49]: We decided to keep home at the bottom of the menu, even though it's not on the designs. Sorry for the confusion

File: springboard_iogt/templates/personae.jinja2 [hunk 2]
```diff
@@ -5,12 +5,14 @@
 <div>
     <div class="h1">{% trans %}Who are you?{% endtrans %}</div>
     {% set query=({'next': request.GET['next']} if 'next' in request.GET else None) %}
+    <ul>
     <div class="personae">
```
Comment (Rizziepit) [inline springboard_iogt/templates/personae.jinja2]: This tag should probably go and `class="personae"` moved to `ul`

File: springboard_iogt/static/css/super.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    #content .home .articles .mariestopes:after {background-color:#e04a5a; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .straighttalk:after {background-color:#c977b2; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .ureport:after {background-color:#f18054; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .ebola:after {background-color:#fdbe61; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+
+    .cta .h2 a {
+        padding: 10px;
+        color: white;
+        display: block;
+        background-color: #28B0B6
```
Comment (Rizziepit) [inline springboard_iogt/static/css/super.css]: It wasn't clear on the original designs, but the background colour must match that of the content section. The colours are [here](https://github.com/universalcore/springboard-iogt/blob/feature/issue-15-add-smart-styling/springboard_iogt/static/css/style.css#L83). I'm attaching a new image on the Trello card that illustrates it.

File: springboard_iogt/static/css/style.css [hunk 4]
```diff
@@ -10,6 +10,7 @@
     html, body { margin: 0; padding: 0; background: #fff; font-size: 16px; font-family: Arial, Helvetica, sans-serif; line-height: 1.4em; color: #666; }
     input, select, textarea { font-size: 95%; font-family: Arial, Helvetica, sans-serif; }
     a { color: #9630c8; text-decoration: none; font-weight: bold; }
+    ul {list-style: none;   margin: 0; padding: 0;}
```
Comment (Rizziepit) [inline springboard_iogt/static/css/style.css]: Maybe put that back to only #submenu?

**Comment by nathanbegbie** [comment]
@Rizziepit listed things are fixed

**Comment by Rizziepit** [comment]
perfect. +1

**Pull request merged**